data/core/doc/init.lua

```diff
@@ -457,7 +457,7 @@ function Doc:replace_cursor(idx, line1, col1, line2, col2, fn)
       self:set_selections(idx, line1, col1, line2, col2)
     end
   end
-  return n
+  return n or 0
```

Comment:
In theory #961 completely fixes this, but for now this is fine.

Reply:
Ohh, I will revert the change I did in favor of what you worked on and review your change